packages/outbox-core/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@message-queue-toolkit/outbox-core",
+  "version": "0.1.0",
```

Comment:
🟢 I'd start at 1.0.0 and follow the semver from there

Reply:
Could we move to `1.0.0` once API gets stable? 

Regarding the release, I thought we could hold on now with releasing to NPM. Once we have prisma adapter, we could do the release.

And then once it's battle tested in our product, let's promote it to 1.0.0. What do you think?

Reply:
makes sense!